README: other resources section

dhimmel opened the pull request:
Reference recent Crossref dumps discussed in #5 and https://github.com/CrossRef/rest-api-doc/issues/271.

Closes https://github.com/greenelab/crossref/issues/5. Let's open new targeted issues for additional discussion.

@de-code and @bnewbold can you review this PR for accuracy?

dhimmel committed:
README: other resources section

Reference recent Crossref dumps discussed in #5.
Closes https://github.com/greenelab/crossref/issues/5

Refs https://github.com/CrossRef/rest-api-doc/issues/271

de-code commented on README.md:
```diff
@@ -74,6 +74,20 @@ conda env create --file=environment.yml
+Daniel Ecer also downloaded the Crossref work metadata in January 2018, using the codebase at [elifesciences/datacapsule-crossref](https://github.com/elifesciences/datacapsule-crossref).
+His database dump is available on [figshare](https://doi.org/10.6084/m9.figshare.5845554.v2 "Crossref Works Dump - January 2018").
+While the multi-part format of this dump is likely less convenient than the dumps produced by this repository, Daniel Ecer's analysis also exports a DOI-to-DOI table of citations/references [available here](https://doi.org/10.6084/m9.figshare.5849916.v1 "Crossref Citation Links - January 2018").
+This citation catalog contains 314,785,304 citations and is thus more comprehensive than the catalog available from [greenelab/opencitations](https://github.com/greenelab/opencitations).
```
Minor correction, they are 314,785,303 citations excluding the header. I wonder whether it would be of interest also to include https://elifesci.org/crossref-data-notebook which provides some stats over the data (this DOI-to-DOI table contains the oa references with DOI minus the duplicates).

de-code reviewed (approved):
Great summary of available resources.

bnewbold commented:
Looks good to me!

I hope Open Citations pulls in the Crossref data in bulk form soon.

dhimmel committed:
Revisions to address comments

Address comments in https://github.com/greenelab/crossref/pull/7#discussion_r166556072
